### PR TITLE
Feat: Create account - Regulated by CQC

### PR DESCRIPTION
**Pull request by aaron-russell**
#### Work done
- Create `new-regulated-by-cqc` component inside new `create-account` folder
- Create new routes `new-regulated-by-cqc`, `find-workplace` and `workplace-name` for registration and parent flow
- Add feature flags to registration and parent start pages to take you to the new page

#### Note
- `find-workplace` and `workplace-name` components haven't been created yet so the URL takes you back to `new-regulated-by-cqc`

#### Tests
Does this PR include tests for the changes introduced?
- [x] Yes
- [ ] No, I found it difficult to test
- [ ] No, they are not required for this change

### CODE REVIEW

File: src/app/features/add-workplace/start/start.component.ts [hunk 1]
```diff
@@ -1,19 +1,36 @@
...
+  createAccountNewDesign: boolean;
+  constructor(
+    private backService: BackService,
+    private workplaceService: WorkplaceService,
+    private featureFlagsService: FeatureFlagsService,
+  ) {}
 
-  ngOnInit() {
+  async ngOnInit(): Promise<void> {
+    this.createAccountNewDesign = await this.featureFlagsService.configCatClient.getValueAsync(
```
Comment (aaron-russell) [inline src/app/features/add-workplace/start/start.component.ts:19]: Do you need to catch any errors here?

File: src/app/features/registration/create-account/create-account.component.ts [hunk 2]
```diff
@@ -1,15 +1,30 @@
...
 })
 export class CreateAccountComponent implements OnInit {
-  constructor(private registrationService: RegistrationService) {}
+  public createAccountNewDesign: boolean;
 
-  ngOnInit(): void {
+  constructor(private registrationService: RegistrationService, private featureFlagsService: FeatureFlagsService) {}
+
+  async ngOnInit(): Promise<void> {
+    this.featureFlagsService.configCatClient.forceRefreshAsync();
```
Comment (aaron-russell) [inline src/app/features/registration/create-account/create-account.component.ts:15]: Same here, do you need catch any errors?

File: src/app/features/add-workplace/start/start.component.ts (hunk 1 above)
Comment (Richard-Pentecost) [reply src/app/features/add-workplace/start/start.component.ts:19]: The `false` statement in the `getValueAsync` call should default the variable to false if there is an error

File: src/app/features/registration/create-account/create-account.component.ts (hunk 2 above)
Comment (Richard-Pentecost) [reply src/app/features/registration/create-account/create-account.component.ts]: see above

**Review by g-m-taylor — APPROVED** [review]

**Pull request merged**